packages/bs-neo3/src/services/vote/RpcVSNeo3.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  CalculateVoteFeeParams,
+  GenerateSigningCallbackResponse,
+  GetCandidatesToVoteResponse,
+  GetVoteDetailsByAddressResponse,
+  VoteParams,
+  VoteResponse,
+  VoteService,
+} from '../../interfaces'
+import { BSNeo3Helper } from '../../helpers/BSNeo3Helper'
+import { NeonInvoker } from '@cityofzion/neon-dappkit'
+import { Account, BSNumberHelper, Network, Token } from '@cityofzion/blockchain-service'
+import { ContractInvocationMulti } from '@cityofzion/neon-dappkit-types'
+import { tx } from '@cityofzion/neon-js'
+import { BSNeo3NetworkId } from '../../constants/BSNeo3Constants'
+
+type GetVoteCIMParams = {
+  address: string
+  candidatePubKey: string
+}
+
+type Params<BSName extends string> = {
+  network: Network<BSNeo3NetworkId>
+  feeToken: Token
+  generateSigningCallback: (account: Account<BSName>) => Promise<GenerateSigningCallbackResponse>
+}
+
+export class RpcVSNeo3<BSName extends string> implements VoteService<BSName> {
```

Comment:
Rename to RpcVoteServiceNeo3. I think these shortenings we made only make things more difficult.

Reply:
Okay, I was trying to follow the project pattern.

Reply:
https://github.com/CityOfZion/blockchain-services/pull/188#discussion_r2136082690